src/main/java/at/hannibal2/skyhanni/features/inventory/experimentationtable/SuperpairsClicksAlert.kt

```diff
@@ -69,7 +69,8 @@ object SuperpairsClicksAlert {
         event.inventoryName.contains("Ultrasequencer") &&
             event.inventoryItems.entries
                 .filter { it.key < 45 }
-                .any { it.value.stackSize > roundsNeeded }
+                // We substract 1 due to a Hypixel bug causing one less round to be required
```

Comment:
Sorry for the back and forth lol, I typoed in my initial review and you applied without seeing the edit, should be the last fix.
```suggestion
                // We subtract 1 due to a Hypixel bug causing one less round to be required
```